Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -88,6 +89,7 @@ jobs:
         id: DetermineWorkflowDepth
         run: |
           Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "WorkflowDepth=$($env:workflowDepth)"
+          Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "ArtifactsRetentionDays=0"
```

Comment:
0 is the default anyway, so why set it explicitly?

Reply:
Yeah - it can be removed (was here because of the longlived setting)